sroka/api/google_ad_manager/gam_api.py

```diff
@@ -92,3 +92,65 @@ def get_data_from_admanager(query, dimensions, columns, start_date, end_date, cu
     with gzip.open(report_file.name) as file:
         data = pd.read_csv(file)
     return data
+
+
+def get_users_from_admanager(query, dimensions, network_code=None):
+    
+    user_df = pd.DataFrame()
+    
+    dimensions_df = pd.DataFrame()
+        
+    #.Where() handles filtering with admanager method, while statement_query handles queries with WHERE clause
+    
+    statement_query = query.replace("WHERE ", "")
+    
+    statement = (ad_manager.StatementBuilder()
+               .Where((statement_query)))
+    
+    if not network_code:
+        try:
+            network_code = config.get_value('google_ad_manager', 'network_code')
+        except (KeyError, NoOptionError):
+            print('No network code was provided')
+            return pd.DataFrame([])
+
+    yaml_string = "ad_manager: " + "\n" + \
```

Comment:
consider concatenating subsequent strings if there is no variable between them. Also consider using .format() function for handling strings variables.

Reply:
Not sure if I follow. All strings here are separated by variables so there is nothing to concatenate? Unless this is about network code.
Re using .format(), do you suggest to replace current way of creating yaml_string with .format() based one?

Reply:
What I mean is `"ad_manager: " + "\n"` can be `"ad_manager: \n"`
As for .format() you could do sth like `"ad_manager: \n application_name: {}\n".format(APPLICATION_NAME)` but this is just a suggestion, we can move on with current solution.

Reply:
Got it, thank you! Let's leave it for now as it is, since this is already tested and I'll apply it in one of next PRs